infoschema/cluster.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package infoschema
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/domain/infosync"
+	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/types"
+)
+
+const (
+	clusterTablePrefix = "TIDB_CLUSTER_"
+)
+
+// Cluster table list, attention:
+// 1. the table name should be upper case.
+// 2. clusterTableName should equal to "TIDB_CLUSTER_" + memTableTableName.
+const (
+	clusterTableSlowLog     = "TIDB_CLUSTER_SLOW_QUERY"  // clusterTablePrefix + tableSlowLog
+	clusterTableProcesslist = "TIDB_CLUSTER_PROCESSLIST" // clusterTablePrefix + tableProcesslist
+)
+
+// memTableToClusterTableMap means add memory table to cluster table.
+var memTableToClusterTableMap = map[string]struct{}{
+	tableSlowLog:     {},
```

Comment:
According to the name, we should map table name to cluster table name.
```go
var memTableToClusterTableMap = map[string]struct{}{
	tableSlowLog:     "TIDB_CLUSTER_SLOW_QUERY",
	tableProcesslist: "TIDB_CLUSTER_PROCESSLIST",
```
And delete the `clusterTablePrefix` definition.

Reply:
Great.